File: Javascript/1010.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var peca1 = lines.shift().split(" ")
var peca2 = lines.shift().split(" ")

var [codigoPeca1, numeroPecasPeca1, valorPeca1]= parseInt(peca1.shift())
var [codigoPeca2, numeroPecasPeca2, valorPeca2]= parseInt(peca1.shift())

var valorTotalPeca1 = numeroPecasPeca1 * valorPeca1

var valorTotalPeca2 = numeroPecasPeca2 * valorPeca2

var valorTotalCompra = valorTotalPeca1 + valorTotalPeca2

console.log("VALOR A PAGAR: R$ " + valorTotalCompra.toFixed(2))

File: Java/1010.java
import java.util.Scanner;

public class Main {

  public static void main(String[] args)  {
    Scanner scanner = new Scanner(System.in);

    int codigoPeca1 = scanner.nextInt();
    int qtdPeca1 = scanner.nextInt();
    double valorPeca1 = scanner.nextDouble();
    
    int codigoPeca2 = scanner.nextInt();
    int qtdPeca2 = scanner.nextInt();
    double valorPeca2 = scanner.nextDouble();
    
    double valorPagar = (qtdPeca1 * valorPeca1) + (qtdPeca2 * valorPeca2);

    String valorPagarFormatado = String.format("%.2f", valorPagar);
    
    System.out.println("VALOR A PAGAR: R$ " + valorPagarFormatado);
  }
}